packages/neo-one-smart-contract-compiler/src/compile/syscalls.ts

```diff
@@ -260,6 +260,7 @@ const OutputValue = new BlockchainInterface('OutputBase');
 const TransactionValue = new BlockchainInterface('TransactionBase');
 const ValidatorValue = new BlockchainInterface('ValidatorBase');
 const StorageContextValue = new BlockchainInterface('StorageContextBase');
+const StorageContextReadonlyValue = new BlockchainInterface('StorageContextReadonlyBase');
```

Comment:
nit: `StorageContextReadOnlyBase` - change `Readonly` to `ReadOnly` everywhere to be consistent with the syscall.